REFERENCES.md

```diff
@@ -0,0 +1,184 @@
+# Scientific Articles
+We encourage you to also check out work by the group behind 
+GluonTS. They are grouped according to topic and ordered 
+chronographically.
+
+## Methods
+A number of the below methods are available in GluonTS.
+
+```
+@inproceedings{salinas2019high,
+	Author = {Salinas, David and Bohlke-Schneider, Michael and Callot, Laurent and Gasthaus, Jan},
+	Booktitle = {Advances in Neural Information Processing Systems},
+	Title = {High-Dimensional Multivariate Forecasting with Low-Rank Gaussian Copula Processes},
+	Year = {2019}
+}
+```
+
+Deep Factor models, a global-local forecasting method.
+```
+@inproceedings{wang2019deepfactors,
+	Author = {Wang, Yuyang and Smola, Alex and Maddix, Danielle and Gasthaus, Jan and Foster, Dean and Januschowski, Tim},
+	Booktitle = {International Conference on Machine Learning},
+	Pages = {6607--6617},
+	Title = {Deep factors for forecasting},
+	Year = {2019}
+}
+```
+DeepAR, an RNN-based probabilistic forecasting model.
+```
+@article{flunkert2019deepar,
```

Comment:
There should be consistency in the tag: is it \<first-author-family-name>\<year>\<first-word-in-title>?

Reply:
yes.

Reply:
Then either the order of the authors or the key must be adjusted here I think?